tests/Integration/TestCase.php

```diff
@@ -106,9 +110,10 @@ public function create_test_blog( $domain, $user_id ) {
 	 *
 	 * @param string $taxonomy_key Taxonomy key, must not exceed 32 characters.
 	 * @param string $term_name    The term name to add.
+	 *
 	 * @return array|WP_Error An array containing the term_id and term_taxonomy_id, WP_Error otherwise.
```

Comment:
How about adding in a try...catch to this method (and similar methods) so that any WP_Error is caught and handle, and letting this method only return the intended `array`?

Reply:
That makes sense, but it is out of the scope of this PR. Will address it separately.